picard/script.py

```diff
@@ -307,6 +308,24 @@ def register_script_function(function, name=None, eval_args=True,
     )
 
 
+_int_operators = {'add': operator.add,
+                  'sub': operator.sub,
+                  'mul': operator.mul,
+                  'div': operator.div,
+                  'mod': operator.mod}
+
+_logic_operators = {'and': (lambda x, y: x and y),
+                    'or': (lambda x, y: x or y)}
+
+
+def _compute_int(operation, *args):
+    return str(reduce(_int_operators[operation], map(int, list(args))))
+
+
+def _compute_logic(operation, *args):
+    return reduce(_logic_operators[operation], list(args))
```

Comment:
You can skip `reduce` here (and `_logic_operators`) and implement this as `return operation(args)` where operation is either `any` or `all`.